boards/cuav/x7pro/src/board_config.h

```diff
@@ -153,8 +153,11 @@
 /* High-resolution timer */
 #define HRT_TIMER               3  /* use timer3 for the HRT */
 #define HRT_TIMER_CHANNEL       3  /* use capture/compare channel 3 */
-#define STM32_RCC_APB1ENR  STM32_RCC_APB1LENR
-#define RCC_APB1ENR_TIM3EN RCC_APB1LENR_TIM3EN
+#define STM32_RCC_APB1ENR  	STM32_RCC_APB1LENR
```

Comment:
Thes3 5 lined does not belong in board_config.h They belongs in the timer driver (preferred) or the micro_hal

Reply:
Thanks for your reply,but could you tell me  the exact file i need to add these 5 lines? I think mocro_hal is a common file ,should i change it ? Look forward to your reply.

Reply:
How about the H7 micro_hal? platforms/nuttx/src/px4/stm/stm32h7/include/px4_arch/micro_hal.h

Then build all the H7 targets and fix them as well.